spec/defines/virtual_machine_spec.rb

```diff
@@ -278,18 +278,10 @@ def contain_preseed
         end
       end
 
-      context 'with build set to jessie' do
-        let(:params) { { build: 'jessie' } }
+      context 'with invalid build name' do
+        let(:params) { { build: 'foobar' } }
```

Comment:
If I revert that, the following happens:

```
# bundle exec rspec spec/classes/profile/vmhost/host_spec.rb
nebula::profile::vmhost::host
  on ubuntu-20.04-x86_64
    is expected to contain File[/etc/default/libvirt-guests]
    when given nothing
      is expected not to contain Nebula::Virtual_machine[vmname]
    when given bridge network interfaces
      is expected to contain Nebula::Virtual_machine[vmname] with internet_bridge => "internet_bridge" (FAILED - 1)
      is expected to contain Nebula::Virtual_machine[vmname] with lan_bridge => "lan_bridge" (FAILED - 2)
...
```

This test makes no sense. We're testing that we can successfully create a vm given an intentionally invalid build. Maybe the bug is in `spec/classes/profile/vmhost/host_spec.rb` rather than `spec/defines/virtual_machine_spec.rb`, but I think this is a bug either way.